### PR TITLE
Jenkins buildbot

### PR DESCRIPTION
**Pull request by slefrancois**

### CODE REVIEW

File: misc/do_nightly_build [hunk 1]
```diff
@@ -1,9 +1,15 @@
 #!/bin/bash
-#we set the compiledir to the /Tmp dir to make the test faster by bypassing the nfs network.
+
+# If not jenkins, set workspace to local Tmp
+if [ -v $WORKSPACE ]; then
+   WORKSPACE=/Tmp
```
Comment (nouiz) [inline misc/do_nightly_build]: This isn't good. Most computer don't have /Tmp. if WORKSPACE isn't defined, and TMPDIR is defined, use TMPDIR else use /tmp.

File: misc/do_nightly_build [hunk 2]
```diff
@@ -19,14 +25,17 @@ echo "git version:" `git rev-parse HEAD`
...
 #THEANO_FLAGS=${FLAGS},mode=FAST_COMPILE ${NOSETESTS}
 echo "executing nosetests speed with mode=FAST_RUN"
-THEANO_FLAGS=${FLAGS},mode=FAST_RUN ${NOSETESTS} test.py:speed
+FILE=${ROOT_CWD}/dlt_tests.xml
+THEANO_FLAGS=${FLAGS},mode=FAST_RUN ${NOSETESTS} ${XUNIT}${FILE} test.py:speed
 #echo "executing nosetests speed with mode=FAST_RUN and OMP_NUM_THREADS=2"
 #OMP_NUM_THREADS=2 THEANO_FLAGS=${FLAGS},mode=FAST_RUN ${NOSETESTS} test.py:speed
 echo "executing nosetests with mode=FAST_RUN,floatX=float32"
-THEANO_FLAGS=${FLAGS},mode=FAST_RUN,floatX=float32 ${NOSETESTS}
+FILE=${ROOT_CWD}/dlt_32bit_tests.xml
```
Comment (nouiz) [inline misc/do_nightly_build]: to make the file name more clear, use dlt_float32_tests.xml

We also test python 32 bit.

**Comment by nouiz** [comment]
Is this ready to merge?

**Comment by slefrancois** [comment]
Good to go.

**Pull request merged**